lib/evmone/baseline.cpp

```diff
@@ -276,7 +323,13 @@ evmc_result execute(const VM& vm, ExecutionState& state, const CodeAnalysis& ana
     if (INTX_UNLIKELY(tracer != nullptr))
```

Comment:
Hm, there are two `if (tracer != nullptr)` expressions here. Any reason for not merging them?

Reply:
See #498.